### ISSUE TITLE
[Discussion]: Write issues, docs and Pr's in english

### ISSUE THREAD

**Issue by valentinpalkovic** [element-plus/element-plus#364]
I would like to give it a try to write issues for element-plus in plain English, that everyone in the world can understand the issue and to avoid duplicate issues. This should be also valid for all written documentation and PR's. In the original Element-UI library, it is nowadays impossible to go through the issues, without understanding mandarin or another Chinese dialect. I think English is the most established language among developers and we should strongly keep this habit. Otherwise, it is a strong argument against element-plus or element-ui to use it in an enterprise business context.

If we decide on this principle, this should be written down into the contribution/issue/pr guidelines.

**Comment by valentinpalkovic** [comment]
@zazzaz What do you think?

**Comment by zazzaz** [comment]
Yes, it will be great if all our issues and pull requests are in English.

And maybe, we could add an issue translation bot to fix those none-English issues.

**Comment by JeremyWuuuuu** [comment]
**Agreed**, even though the population of developers in China has been raising for the past years, the majority of those who speaks Chinese were not willing to use English on Github, which to us who understands Chinese is not a problem but for those who does not, will be a big problem.

I **agree** with @zazzaz that we should import the `Translation Bot` into this repo that users/developers do not speak Chinese could understand the context on those issues/PRs. One thing for sure is that we took this community very seriously, we want the project to be a world class project.

**Comment by JeremyWuuuuu** [comment]
We will encourage users to use English in this project. Also we have made our translation bot working. So I am closing this issue. Thank you for making this discussion.
